docs/components/checkbox-field/demo/base-demo.md

```diff
@@ -3,7 +3,10 @@
   @label='Label'
   @isChecked={{this.isChecked}}
   @onChange={{this.handleChange}}
-/>
+>
+  <:label>extra label info</:label>
+  <:hint>extra hint info</:hint>
```

Comment:
For this you could maybe add a `<a href="https://www.crowdstrike.com">Learn more</a>` with some additional styling to make it look like a Toucan link